cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -130,14 +143,33 @@ public static class AesGcmSimple
 
 ```
 
+</details>
+
+_**End of code snippet.**_
```

Comment:
Do we need this? It seems extraneous.

Reply:
So the problem is that I am not sure how clearly the collapsible sections will render so I want to make sure it is clear where the code starts and ends @szh

Reply:
@tghosth Is it not going to look like it does in the Github preview?

Reply:
So it looks different in the real cheatsheet site to how it looks in Github preview and [actually looks a lot better](https://cheatsheetseries.owasp.org/cheatsheets/Java_Security_Cheat_Sheet.html#symmetric-example-using-google-tink) so I am going to remove the extra comment in a new PR

Reply:
See #1162 @szh